src/buildmanager.cpp

```diff
@@ -1495,6 +1495,18 @@ void BuildManager::checkLatexConfiguration(bool &noWarnAgain)
 	}
 }
 
+#ifdef Q_OS_WIN32
+void BuildManager::swapWinSearchPath(QString &miktex, QString &texlive)

```

Comment:
not sure that this special method is necessary useful.
You should have direct access to the miktexBinPath... (and the name is confusing: swap usually means exchanging the both given variables)

Reply:
The purpose is, if a MiKTeX version was detected, to change the variable to the desired target path only for the duration of the manual search. After the search the path is changed back again (your second comment). This is helpful if you have installed a MiKTeX version regularly and are working with a portable version in parallel.